doc/sphinx/proof-engine/tactics.rst

```diff
@@ -2939,6 +2939,77 @@ simply :g:`t=u` dropping the implicit type of :g:`t` and :g:`u`.
       make some proof steps explicit when refactoring a proof script
       to make it readable.
 
+   .. tacv:: convert_concl_no_check @term
+      :name: convert_concl_no_check
+
+      For advanced usage. Similar to :n:`change @term`, but as an optimization,
+      it skips checking that :n:`@term` is convertible to the goal.
+
+      Recall the Coq kernel typechecks proofs again when they are concluded to
+      ensure safety. Hence, using :tacn:`change` checks convertibility twice
+      overall, while :tacn:`convert_concl_no_check` can produce ill-typed terms,
+      but checks convertibility only once.
+      Hence, :tacn:`convert_concl_no_check` can be useful to speed up certain proof
+      scripts, especially if one knows by construction that the argument is
+      indeed convertible to the goal.
+
+      In the following example, :tacn:`convert_concl_no_check` replaces :g:`False` by
+      :g:`True`, but :g:`Qed` then rejects the proof, ensuring consistency.
+
+      .. example::
+
+         .. coqtop:: all abort
+
+            Goal False.
+              convert_concl_no_check True.
+              exact I.
+              Fail Qed.
+
+.. tacn:: exact_no_check @term
```

Comment:
The location of this documentation doesn't seem right. If you want to have all the `no_check` tactic be documented close together, then you could create a new section for these tactics. But here, you chose to document them in the section on reduction tactics, which only makes sense for `convert_concl_no_check`.